vllm/v1/worker/tpu_model_runner.py

```diff
@@ -746,7 +755,9 @@ def capture_model(self) -> None:
         start = time.perf_counter()
         num_tokens = 16
         while True:
-            self._dummy_run(self.kv_caches, num_tokens)
+            with self.maybe_profile_with_lora(
```

Comment:
where can I find this function?

Reply:
That should be [here](https://github.com/krai/vllm/blob/main/vllm/v1/worker/lora_model_runner_mixin.py#L86). The name changed in main a few hours ago to `maybe_dummy_run_with_lora`